src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Overview/DiaryOverviewTableViewController.swift

```diff
@@ -115,15 +122,15 @@ class DiaryOverviewTableViewController: UITableViewController {
 		tableView.estimatedRowHeight = 60
 	}
 
-	private func descriptionCell(at indexPath: IndexPath) -> UITableViewCell {
+	private func descriptionCell(forRowAt indexPath: IndexPath) -> UITableViewCell {
 		guard let cell = tableView.dequeueReusableCell(withIdentifier: String(describing: DiaryOverviewDescriptionTableViewCell.self), for: indexPath) as? DiaryOverviewDescriptionTableViewCell else {
 			fatalError("Could not dequeue DiaryOverviewDescriptionTableViewCell")
```

Comment:
FatalError -> better Log.error and return an empty UITableViewCell

Reply:
See above